criu/cr-service.c

```diff
@@ -520,6 +520,9 @@ static int setup_opts_from_req(int sk, CriuOpts *req)
 		case CRIU_NETWORK_LOCK_METHOD__NFTABLES:
 			opts.network_lock_method = NETWORK_LOCK_NFTABLES;
 			break;
+		case CRIU_NETWORK_LOCK_METHOD__NONE:
```

Comment:
you might need to also update the `criu_network_lock_method` enum in `rpc.proto`:
https://github.com/checkpoint-restore/criu/blob/c75c017e/images/rpc.proto#L52

Reply:
Done.